docs/command-reference/sets/saddex.md

```diff
@@ -19,7 +19,8 @@ import PageTitle from '@site/src/components/PageTitle';
 **Warning:** Experimental! Dragonfly-specific.
 
 Similar to [`SADD`](sadd.md) but adds one or more members that expire after specified number of seconds.
-An error is returned when the value stored at `key` is not a set.
+An error is returned when the value stored at `key` is not a set. If the `KEEPTTL` option is specified, any
```

Comment:
Please separate to different lines , the error code explained should be last

Reply:
How about
By default, the command updates the TTL for an existing member. However, when the optional KEEPTTL parameter is used, the member's original time-to-live remains unchanged.

Reply:
Thanks, I split and reordered the content, does it look okay now?